website/content/docs/commands/pki/health-check.mdx

```diff
@@ -325,6 +447,16 @@ This health check verifies that tidy has run within the last run window. This ca
 
 This health check verifies that this cluster has a reasonable number of certificates. Ideally this would be fetched from tidy's status or a new metric reporting format, but as a fallback when tidy hasn't run, a list operation will be performed instead.
 
+**Remediation Steps**:
+
+If a large amount of certificates have accumulated, verify that tidy has recently run using `vault read <mount>/tidy-status`. Consider
+enabling `auto-tidy` based on instructions above.
+
+Also ensure that certificates are not being renewed too soon based on lifetimes, roughly 2/3 of its lifetime, and
+certificates are being issued with appropriate lifetimes based on their usage.
+
+To help going forward, consider setting all role's `no_store` field to `true` and leverage [BYOC revocations](/vault/api-docs/secret/pki#revoke-certificate) to avoid storage.
```

Comment:
```suggestion
To help going forward, consider setting all roles' `no_store` fields to `true` and leverage [BYOC revocations](/vault/api-docs/secret/pki#revoke-certificate) to avoid storage.
```